### DISCUSSION TITLE
Roll ups

### DISCUSSION THREAD

**Discussion by srikarthickeyanganapathy** [Cyfrin/foundry-full-course-cu#2537] [Q&A] [answered]
I want to know, what is a roll up and key difference between optimistic and ZK

**Comment by EngrPips** [comment, marked as answer]
Hello @srikarthickeyanganapathy, This article might be a good start https://cointelegraph.com/explained/what-are-blockchain-rollups

**Reply by srikarthickeyanganapathy** [reply]
@EngrPips thanks a million. This article can help me understand what is a rollup and the two types of rollup's

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
